src/js/components/Transaction/TransferForm.js

```diff
@@ -163,7 +162,7 @@ const TransferForm = (props) => {
       </div>
 
       {props.account !== false &&
-        <PostTransfer/>
+        <PostTransfer destAddress={props.input.destAddress.value}/>
```

Comment:
Why you dont pass destAddress.value to global ?

Reply:
It should be a state of a component because it can either be ETH name address or a normal address.